engine/pom.xml

```diff
@@ -312,10 +312,6 @@
       ${camunda.artifact}.container.*;${camunda.osgi.version};-noimport:=true,
       ${camunda.artifact}.engine.*;${camunda.osgi.version};-noimport:=true
     </camunda.osgi.export.pkg>
-    <camunda.osgi.import.defaults>
-      org.camunda.bpm.engine.variable.*,
-      org.camunda.commons.*;${camunda.osgi.import.camunda.commons.version},
```

Comment:
If this is removed, what version will packages with `org.camunda.commons.*` receive? I guess we need to keep this line in the defaults and only remove the `org.camunda.bpm.engine.variable.*`, don't we?

Reply:
You will get this version
```
org.camunda.commons.logging;version="[1.10,2)"
org.camunda.commons.utils.cache;version="[1.10,2)"
org.camunda.commons.utils;version="[1.10,2)"
```

Reply:
Thanks for that quick input! OK, that would be fine with me then. 👍